.pre-commit-config.yaml

```diff
@@ -0,0 +1,8 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.2.0
+    hooks:
+    -   id: check-yaml
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
+        exclude: ^.*\.(pdb|cif|fasta)$
```

Comment:
What is the reason for this exclusion?

Reply:
Great question: many files downloaded from the PDB are right-padded with whitespaces. This pre-commit would strip all of these away, creating a big diff that differs from the source of truth (the PDB).